code/modules/mob/mob.dm

```diff
@@ -587,6 +587,7 @@
 /mob/proc/facedir(ndir)
 	if(!canface())
 		return FALSE
+	SEND_SIGNAL(src, COMSIG_MOB_FACE_DIR, ndir)
```

Comment:
delete

Reply:
why ? this is used so it is player inputted to change the dir and not changed by everything else like *spin and other stuff, causing people to *vomit

Reply:
I mean if tehy spin with a scope and make themselves vomit sounds like a personal issue tbh